extensions/podman/src/extension.ts

```diff
@@ -198,6 +200,24 @@ async function monitorMachines(provider: extensionApi.Provider) {
   }
 }
 
+// Monitor the information and warning messages of the provider
+async function monitorInformationAndWarnings(provider: extensionApi.Provider) {
+  // Call again until we receive "stopLoop"
+  if (!stopLoop) {
+    try {
+      // Grab the information and warning messages
+      provider.updateInfo(await getInformation());
+      provider.updateWarning(await getWarning());
+    } catch (error) {
+      // ignore the update
+    }
+
+    // Wait 5 seconds as we do not want to call too often
+    await timeout(5000);
+    monitorInformationAndWarnings(provider);
+  }
+}
+
```

Comment:
do we need this infinite loop ? or we need to track specific event like if `/var/run/docker.sock` file is modified ?

Reply:
Yup, we need the infinite loop in order to retrieve any new information as it's being retrieved via `getInformation` and `getWarning`.

Should we increase the timer though to maybe 10 seconds?

Reply:
I don't see what values are not event based for podman provider ? (this is why I would say we don't need loop but filesystem event)
it seems it's only tracking the socket

Reply:
we have ways to be notified when some files are changing so we could update info and warning only if the socket file is being updated (info is empty for now) and warning is based on the socket path

Reply:
Actually, I should move this into `monitorProvider`. Let me refactor.

Reply:
but we don't need to do that on a periodic schedule, this is what I'm trying to explain.
it's event based